Jenkinsfile

```diff
@@ -131,6 +132,20 @@ def reboot(){
     build job: 'reboot-slaves', propagate: false , parameters: [string(name: 'server', value: "${env.NODE_NAME}"),]
 }
 
+def tensileStage(cmd, gpu_arch, miotensile_version, target_id){
+    try{
+        buildHipClangJob('/opt/rocm/llvm/bin/clang++', image: 'miopen-hip-clang-targetid', cmd: cmd, gpu_arch: gpu_arch, miotensile_version: miotensile_version, target_id: target_id)
```

Comment:
I'm not quite sure about this. Paul requested miopentensile not being built in regular stages, which means the regular and miopentensile stage will adopt different docker images. If we remove the image name, then we will need to rebuild image every time instead of taking advantage of the built docker.

Reply:
We rebuild image every time anyway. Usually this is super fast, because docker takes layers from the cache. Image name does not matter.

Reply:
OK, I will address your comment after CI is back.